src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -24896,9 +25177,7 @@ static void F(object? o)
                 Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "(new { P = o }).P").WithLocation(5, 9));
         }
 
-        // https://github.com/dotnet/roslyn/issues/29891: NullableWalker.VisitAnonymousObjectCreationExpression
-        // should support initializers with inferred nullability.
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/29891")]
+        [Fact]
```

Comment:
>Fact [](start = 9, length = 4)

Can you test a missing method on anonymous type? `(new { P = o }).Missing()`
Link to [https://github.com/dotnet/roslyn/issues/33565](https://github.com/dotnet/roslyn/issues/33565) is crash